tasks.py

```diff
@@ -14,8 +14,13 @@ def create_database(ctx):
     db_password = os.environ.get("POSTGRES_PASSWORD")
 
     # Establishing connection to PostgreSQL server
+    # (connect to initial database 'postgres' to create a new database)
     conn = psycopg2.connect(
-        host=db_host, port=db_port, user=db_user, password=db_password
+        host=db_host,
+        port=db_port,
+        dbname="postgres",
```

Comment:
maybe I am missing something but why do you not just do:
```
dbname=db_name,
```
since on line 12 you already get that in 
`db_name = os.environ.get("POSTGRES_DB")`

Reply:
that's because it may not exist yet, and 'postgres' should probably always exist. So in a first step we just connect to default 'postgres' and then we can create the database that we want to use.

I acknowledge that in devops setup `POSTGRES_DB` may already exist so we could connect to `db_name = os.environ.get("POSTGRES_DB")` already, but in test environments that's not always the case. I think this way it should work fine for all environments (deployments and test envs).